indico/modules/events/sessions/templates/management/_session_list.html

```diff
@@ -0,0 +1,110 @@
+                            <input class="select-row" type="checkbox" value="{{ session.id }}">
+                        </td>
+                        <td class="i-table">#{{ session.friendly_id }}</td>
+                        <td class="i-table"><strong>{{ session.title }}</strong></td>
+                        <td class="i-table">{{ session.code }}</td>
+                        <td class="i-table">
+                            {%- if session.is_poster %}
+                                {%- trans %}Poster{% endtrans -%}
+                            {% else %}
+                                {%- trans %}Standard{% endtrans -%}
+                            {% endif -%}
+                        </td>
+                        <td class="i-table">
+                            <div class="group">
+                                <span class="js-color-switch"
+                                      data-colors="{{ default_colors | tojson | forceescape }}">
+                                    <a class="i-button icon-palette switch-trigger"
+                                       style="background: #{{ session.background_color }};
+                                              color: #{{ session.text_color }} !important;"
+                                       data-method="PATCH" data-href="{{ url_for('.modify_session', session) }}">
+                                    </a>
+                                </span>
+                            </div>
+                        </td>
+                        <td class="i-table">
+                            <span class="i-label show-session-blocks">
+                                {%- trans %}Session blocks {% endtrans -%}
+                                <span class="badge">{{ session.blocks | length }}</span>
+                            </span>
+                        </td>
+                        <td class="i-table">
+                            <div class="group right">
+                                <a class="icon-edit highlight edit-session action-icon"
+                                   title="{% trans %}Edit this session{% endtrans %}"
+                                   data-title="{% trans title=session.title %}Edit session {{ title }}{% endtrans %}"
+                                   data-href="{{ url_for('.modify_session', session) }}"></a>
+                                <a class="icon-users highlight session-persons-list"
+                                   data-title="{% trans title=session.title %}List of session persons{% endtrans %}"
+                                   data-href="#"></a>
+                                <a class="icon-time highlight action-icon"
+                                   title="{% trans %}Go to this session timetable{% endtrans %}"
+                                   data-href="#"
+                                   data-method="POST"></a>
+                                <a class="icon-remove highlight js-remove-session action-icon"
```

Comment:
Check if you can avoid altogether `js-remove-session` by using `data-update`. I see you re-init the color pickers, but maybe they can trigger them by registering a listener on the container that won't get re-rendered by `data-update`.